config/manifests/inferencepool-resources.yaml

```diff
@@ -1,3 +1,6 @@
+# Note: If you change this file, please also change the file used for e2e tests!
+# 
+# https://github.com/rramkumar1/gateway-api-inference-extension/blob/main/test/testdata/inferencepool-e2e.yaml
```

Comment:
can you fix the path to point to public repo instead of private fork?

Reply:
Done